Cargo.toml

```diff
@@ -1,36 +1,27 @@
 [package]
 name = "tree-sitter-c-sharp"
-description = "C# grammar for the tree-sitter parsing library"
-version = "0.20.0"
+description = "C# grammar for tree-sitter"
+version = "0.21.0"
 authors = [
-    "Damien Guard <damieng@gmail.com>",
-    "Max Brunsfeld <maxbrunsfeld@gmail.com>",
-    "Martin Midtgaard <martin.midtgaard@gmail.com>",
-    "Sjoerd Langkemper <sjoerd-github@linuxonly.nl>",
-    "Patrick Thomson <patrickt@github.com>",
-    "Noelle Caldwell <noelle.caldwell@microsoft.com>",
-    "Douglas Creager <dcreager@dcreager.net>",
+  "Max Brunsfeld <maxbrunsfeld@gmail.com>",
```

Comment:
Why are we wiping some authors out here?

Reply:
I consider the Cargo.toml authors list to be those that are relevant to the **Rust** crate and core grammar, and can be reached out to for any questions, comments, or concerns. Some of those authors either a) have no direct relation to the rust crate, b) are potentially unreachable/inactive.